internal/hud/renderer.go

```diff
@@ -94,7 +95,7 @@ var podStatusColors = map[string]tcell.Color{
 	"CrashLoopBackOff":  cBad,
 }
 
-func (r *Renderer) layout(v view.View) rty.Component {
+func (r *Renderer) layout(v view.View) (c rty.Component, selectedResource view.Resource) {
```

Comment:
not positive about the best way to find/store the selected resource... seems wrong to store it on the render? this is the best i've got -- does this seem right to ppl?

Reply:
Argh, I'm sorry. Yeah, we should rethink the structure of how to store state after a render.